docs/pvacseq/output_files.rst

```diff
@@ -172,80 +190,144 @@ all_epitopes.tsv and filtered.tsv Report Columns
+filtered.tsv.reference_matches Report Columns
+---------------------------------------------
+
+This file is only generated when the ``--run-reference-proteome-similarity``
+option is chosen.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Column Name
+     - Description
+   * - ``Chromosome``
+     - The chromosome of this variant
+   * - ``Start``
+     - The start position of this variant in the zero-based, half-open coordinate system
+   * - ``Stop``
+     - The stop position of this variant in the zero-based, half-open coordinate system
+   * - ``Reference``
+     - The reference allele
+   * - ``Variant``
+     - The alt allele
+   * - ``Transcript``
+     - The Ensembl ID of the affected transcript
+   * - ``Peptide``
+     - The peptide sequence submitted to BLAST
+   * - ``Hit ID``
+     - The BLAST alignment hit ID (reference proteome sequence ID)
+   * - ``Hit Definition``
+     - The BLAST alignment hit definition (reference proteome sequence name)
+   * - ``Query Sequence``
+     - The BLAST query sequence
+   * - ``Match Sequence``
+     - The BLAST match sequence
+   * - ``Match Start``
+     -  The match start position in the matched reference proteome sequence
+   * - ``Match Stop``
+     - The match stop position in the matched reference proteome sequence
+
+.. _aggregated:
+
+all_epitopes.aggregated.tsv Report Columns
 --------------------------------------------
 
+The ``all_epitopes.aggregated.tsv`` file is an aggregated version of the all_epitopes TSV.
+It presents the best-scoring (lowest binding affinity)
+epitope for each variant, and outputs additional binding affinity, expression, and
+coverage information for that epitope. It also gives information about the
+total number of well-scoring epitopes for each variant, the number of
+transcripts covered by those epitopes, as well as the HLA alleles that those
+epitopes are well-binding to. Lastly, the report will bin variants into tiers
+that offer suggestions as to the suitability of variants for use in vaccines.
+
 .. list-table::
    :header-rows: 1
 
    * - Column Name
      - Description
-   * - ``Gene Name``
-     - The Ensembl gene name of the affected gene.
-   * - ``Mutation``
-     - The amino acid change of this mutation.
-   * - ``Protein Position``
-     - The protein position of the mutation.
-   * - ``HGVSc``
-     - The HGVS coding sequence name.
-   * - ``HGVSp``
-     - The HGVS protein sequence name.
-   * - ``HLA Allele``
-     - The HLA allele for this prediction.
-   * - ``Mutation Position``
+   * - ``HLA Alleles`` (multiple) (T/F)
+     - For each HLA allele in the run, did the mutation result in an epitope that binded well
+       to the HLA allele (median mutant binding affinity < 1000).
+   * - ``Gene``
+     - The Ensembl gene name of the affected gene
+   * - ``AA_change``
+     - The amino acid change for the mutation
+   * - ``Num_Transcript``
+     - The Number of transcripts for this mutation that resulted in at least
```

Comment:
The Number of transcripts for this mutation that resulted in at least... -> The number of transcripts for this mutation that resulted in at least...